packages/react/src/components/Combobox/Combobox.test.tsx

```diff
@@ -1638,3 +1664,223 @@ test('should have no axe violations with no matching results', async () => {
   const results = await axe(comboboxRef.current!);
   expect(results).toHaveNoViolations();
 });
+
+test('should render combobox with description', () => {
+  render(
+    <Combobox label="label" description="description">
+      <ComboboxOption>Apple</ComboboxOption>
+      <ComboboxOption>Banana</ComboboxOption>
+      <ComboboxOption>Cantaloupe</ComboboxOption>
+    </Combobox>
+  );
+  expect(screen.getByText('description')).toBeInTheDocument();
```

Comment:
This just validates that the text is in the document, but not necessarily that it's associated with the field. We should have a test to check for accessible description as well, similar to this test: https://github.com/dequelabs/cauldron/blob/d985ed72b148d58a6031ddb52dd5aed5ecbbc426/packages/react/src/components/TextField/TextField.test.tsx#L66-L73